src-tauri/src/lib.rs

```diff
@@ -288,6 +311,27 @@ pub fn run() {
         ])
         .setup(|app| {
             *APP_HANDLE.write().unwrap() = Some(app.handle().to_owned());
+            println!("Setting up yay!");
+
+            std::thread::spawn(move || {
+                // Simulate emitting a message after a delay
```

Comment:
Why are we waiting?

Reply:
We wait in case the frontend starts slightly after the backend. We, don't want the frontend to miss the first term update.